tests/foundry/client/pixi/perception/clockwise-sweep.test-d.ts

```diff
@@ -1,38 +1,16 @@
+import { Point } from "pixi.js";
 import { expectTypeOf } from "vitest";
 
-const someRay = new Ray({ x: 0, y: 0 }, { x: 0, y: 0 });
-const somePolygonRay: PolygonRay = someRay as PolygonRay;
-somePolygonRay.result = new CollisionResult();
+const pointA = new Point(0, 0);
+const pointB = new Point(0, 0);
+// Genuinely confused what's going on here
+// without this I get 'Arguments for the rest parameter 'MISMATCH' were not provided.'; possible bug in vitest?
+const testResult = {} as never;
```

Comment:
So this error has unfortunate DX but what it's saying is that an `expectTypeOf(...).toEqualTypeOf<...>();` failed.

By giving it a parameter you're simply working around the error, not actually fixing the underlying issue. In other words `ClockwiseSweepPolygon.testCollision(pointA, pointB, { mode: "any" })` is NOT `boolean`. In fact it seems to be `boolean | PolygonVertex | PolygonVertex[] | null;` based upon my search for the typing of `testCollision`. Please remove this as it's basically asserting that the type are NOT equal right now.